binding/web/src/orca.ts

```diff
@@ -270,157 +251,148 @@ export class Orca {
   }
 
   /**
-   * Synthesizes a string of text.
+   * Generates audio from text. The returned audio contains the speech representation of the text.
    *
-   * @param text A frame of audio with properties described above.
+   * @param text Generates audio from text. The returned audio contains the speech representation of the text.
    * The maximum number of characters per call to `.synthesize()` is `.maxCharacterLimit`.
    * Allowed characters are lower-case and upper-case letters and punctuation marks that can be retrieved with `.validCharacters`.
    * Custom pronunciations can be embedded in the text via the syntax `{word|pronunciation}`.
    * The pronunciation is expressed in ARPAbet format, e.g.: "I {live|L IH V} in {Sevilla|S EH V IY Y AH}".
-   * @param speechRate Optional argument to configure the rate of speech of the synthesized audio.
+   * @param speechRate Optional argument to configure the rate of speech of the synthesized speech.
    */
-  public async synthesize(text: string, speechRate: number = 1.0): Promise<void> {
+  public async synthesize(text: string, speechRate: number = 1.0): Promise<Int16Array> {
```

Comment:
For now we only add `speechRate` as the only options for synthesize, pretty sure I heard this might change continuously in the future. I think we pass this as something similar to init where we pass in an object for synthesize configurations.